crates/chia-protocol/src/program.rs

```diff
@@ -407,15 +415,7 @@ impl Program {
         }
     }
 
-    fn to_program<'a>(&self, py: Python<'a>) -> PyResult<Bound<'a, PyAny>> {
-        use std::rc::Rc;
-        let mut a = Allocator::new_limited(500_000_000);
-        let prg = node_from_bytes_backrefs(&mut a, self.0.as_ref())?;
-        let prg = LazyNode::new(Rc::new(a), prg);
-        to_program(py, prg)
-    }
-
-    fn uncurry<'a>(&self, py: Python<'a>) -> PyResult<(Bound<'a, PyAny>, Bound<'a, PyAny>)> {
+    fn uncurry_to_lazynode(&self) -> PyResult<(LazyNode, LazyNode)> {
         use clvm_utils::CurriedProgram;
         use std::rc::Rc;
```

Comment:
If you're hoisting the `use` statements out to the file-scope, I think you should remove this one as well

Reply:
Done